Cargo.toml

```diff
@@ -25,7 +25,7 @@ byteorder = "1"
 log = "0.4.8"
 lazy_static = "1.4.0"
 ocl = { version = "0.19.4", package = "fil-ocl", optional = true }
-itertools = { version = "0.8.0", optional = true }
+ff-cl-gen = { git = "https://github.com/filecoin-project/ff-cl-gen", optional = true }
```

Comment:
lets make a release of that before merging, as we can't publish bellperson to crates.io otherwise

Reply:
hmm, can I release that from my personal account too? or should we do it with Filecoin's?

Reply:
you can release it from your personal account and then give @cryptonemo and myself at least access on crates.io